src/app/activity/[id]/components/add-favorite/add-favorite.component.tsx

```diff
@@ -0,0 +1,20 @@
+import { ReactElement } from "react";
+
+import ButtonComponent from "@/components/button/button.component";
+
+import HugeiconsFavourite from "@/icons/HugeiconsFavourite";
+
+import styles from "./add-favorite.module.css";
+
+export default function AddFavoriteComponent(): ReactElement {
+  return (
+    <ButtonComponent
+      shape={"ghost"}
+      variant={"muted"}
```

Comment:
وقتی String معمولی داریم، نیازی به آکلاد نیست

```suggestion
      shape="ghost"
      variant="muted"
```